src/utils/tst/MathTest.cpp

```diff
@@ -0,0 +1,12 @@
+#include "UtilTestFixture.h"
+
+class MathTest : public UtilTestBase {
+};
+
+TEST_F(MathTest, testPowerAPI)
```

Comment:
This test suite is missing overflow tests

Reply:
Intentionally not written those tests as we know there will be an overflow and we have not handled it. As explained above, I am biased towards keeping it simple.